src/plugins/telemetryTable/TelemetryTable.js

```diff
@@ -102,8 +103,8 @@ define([
         }
 
         createTableRowCollections() {
-            this.boundedRows = new BoundedTableRowCollection(this.openmct);
-            this.filteredRows = new FilteredTableRowCollection(this.boundedRows);
+            this.sortedRows = new SortedTableRowCollection(this.openmct);
```

Comment:
I don't think this is necessary at all any more? The `FilteredTableRowCollection` extends `SortedTableRowCollection`, so it handles user sorting AND filtering. As I mentioned in another comment, the TelemetryCollection needs to maintain telemetry sorted ascending anyway, so I can't see why we would need a third, sorted telemetry collection?

Reply:
Agreed, once I update TelemetryCollection to handle that, I'll remove this as well.

Reply:
I've narrowed this down to one table collection now, i'm handling telemetry collections in telemetry rows now.